src/stateful-event-subscriber.ts

```diff
@@ -173,6 +181,10 @@ export abstract class StatefulEventSubscriber<State>
     this.isInitialized = true;
   }
 
+  protected getValueToPushIntoNewPoolsChannel() {
+    return {};
```

Comment:
please make it abstract or throw an error

Reply:
@mwamedacen I didn't get your point here. Why we need to make it abstract? Now getValueToPushIntoNewPoolsChannel method contains default implementation for all pools.

Reply:
updated